### PR TITLE
Adding Markers to Map

### PR DESCRIPTION
**Pull request by lqueipom**

### CODE REVIEW

**Comment by LukeGMTravis** [comment]
Starting review

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -126,6 +127,16 @@ function visualizeMap() {
 
     {name: 'Styles Map'});
 
+  var myHome = {
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:130]: It's preferred to use `const` if there's no reassignment

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -137,8 +148,34 @@ function visualizeMap() {
       mapTypeControlOptions: {
         mapTypeIds: ['hybrid', 'styles_map']
       }
-    });
+  });
+  
+  //Marker positioned at my house. 
+  var markerHome = new google.maps.Marker({
+    position: myHome,
+    map: map,
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:156]: This is perfectly fine but I wanted to note that technically you don't have to say `map: map,` here, you can simplify it to just `map,` if you'd prefer. (This is a shortcut JS allows when they're the same name.)

Comment (lqueipom) [reply portfolio/src/main/webapp/script.js]: Ohhh that's cool! Just made the two changes that you suggested, let me know if there is anything else I should change! :)

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -74,6 +74,7 @@ function createListElement(text) {
 }
 
 function visualizeMap() {
+  // Establishes the styles I want added to my map. 
   var stylesMap = new google.maps.StyledMapType(
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:78]: Can you also change this to const (same reason)

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -137,8 +148,34 @@ function visualizeMap() {
       mapTypeControlOptions: {
         mapTypeIds: ['hybrid', 'styles_map']
       }
-    });
+  });
+  
+  //Marker positioned at my house. 
+  var markerHome = new google.maps.Marker({
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:154]: Ditto

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -137,8 +148,34 @@ function visualizeMap() {
...
+    position: myHome,
+    map,
+    title: 'My house!'
+  });
 
+  // Marker positioned at Maracaibo Lake.
+  var markerLake = new google.maps.Marker({
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:161]: Ditto

Comment (lqueipom) [reply portfolio/src/main/webapp/script.js]: Perfect, thanks for the advice will keep it in mind for the future :) Fixed the issues, should be okay now!

**Pull request merged**